_content/av/back-to-nature_yuttadhamo.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Back to nature"
+authors:
+- yuttadhammo
+external_url: "https://tunein.com/podcasts/Religion--Spirituality-Podcas/Dhamma-Talks-(Part-3)-p2926505/?topicId=220379444"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1cmOj8zVQPfMFeLqjq_ugnOf5-bhzoki4/view?usp=sharing"
+formats: [m4a]
+course: nature
+tags:
+  - meditation
+  - city
+  - mind
```

Comment:
Hmmm maybe tag with its opposite: `materialism` as it's largely a reaction to that.